src/components/data-chart/index.tsx

```diff
@@ -0,0 +1,471 @@
+import {
+  Button,
+  CircularProgress,
+  Dialog,
+  Divider,
+  Icon,
+  Scrim,
+  Typography,
+} from '@equinor/eds-core-react';
+import { clear } from '@equinor/eds-icons';
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { Chart } from 'react-google-charts';
+import { getJson } from '../../dynatrace-api/api-helpers';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { configVariables } from '../../utils/config';
+import './style.css';
+
+const AvailabilityCharts = () => {
```

Comment:
remember to type functions and variables

for example:
```suggestion
const AvailabilityCharts = (): JSX.Element => {
```